server/tests/2-recipes.js

```diff
@@ -584,7 +586,7 @@ describe('/GET all Recipes Test', () => {
       .set('Accept', 'application/json')
       .set('x-access-token', token)
       .end((err, res) => {
-        expect(res.statusCode).to.equal(201);
+        expect(res.statusCode).to.equal(200);
```

Comment:
Your test should cover other aspects of your payload not just the status code of the response